<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PosWindow</class>
 <widget class="QMainWindow" name="PosWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="login">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="posPrincipal">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>771</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridMesas"/>
     </widget>
    </widget>
    <widget class="QWidget" name="menu">
     <widget class="QStackedWidget" name="stackedCategorias">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>551</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageCategoria">
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>551</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridCategoria"/>
       </widget>
      </widget>
      <widget class="QWidget" name="pageItems">
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>541</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridItems"/>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="enviarButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enviar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pagarButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagar</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>191</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutPedidos"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
